endpoints/api/billing.py

```diff
@@ -998,14 +1000,14 @@ def post(self, orgname):
 @resource("/v1/organization/<orgname>/marketplace/<subscription_id>")
 @path_param("orgname", "The name of the organization")
 @path_param("subscription_id", "Marketplace subscription id")
-@related_user_resource(UserPlan)
 @show_if(features.BILLING)
 class OrganizationRhSkuSubscriptionField(ApiResource):
     """
     Resource for removing RH skus from an organization
     """
 
     @require_scope(scopes.ORG_ADMIN)
+    @nickname("removeSkuFromOrg")
```

Comment:
Is there currently a way to swap available SKU from an org?

Reply:
Also, if I want to stack multiple SKUs, should I use the `POST` endpoint?

Reply:
There's no way to directly swap at the moment, you would need to remove the SKU from the org and assign it somewhere else. Multiple `POST` requests will stack the SKUs